hack/kind-setup-loopback-devices.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+MULTI_ZONAL="false"
+CLUSTER_NAME=""
+IPFAMILY="ipv4"
+
+SUDO=""
+if [[ "$(id -u)" != "0" ]]; then
+  SUDO="sudo "
+fi
+
+parse_flags() {
+  while test $# -gt 0; do
+    case "$1" in
+    --cluster-name)
+      shift; CLUSTER_NAME="${1}"
+      ;;
+    --ip-family)
+      shift; IPFAMILY="${1}"
+      ;;
+    --multi-zonal)
+      MULTI_ZONAL=true
+      ;;
+    esac
+
+    shift
+  done
+}
+
+parse_flags "$@"
+
+
+
+LOOPBACK_IP_ADDRESSES=(172.18.255.1)
```

Comment:
Nit: Usually, we do not leave several empty lines.

```suggestion
parse_flags "$@"

LOOPBACK_IP_ADDRESSES=(172.18.255.1)
```

Reply:
Thanks for the review. I think these were some accidental enters that I pressed 😅